src/skin/qml/qmlconfigproxy.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <QColor>
+#include <QObject>
+#include <QVariantList>
+
+#include "preferences/usersettings.h"
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+
+class QmlConfigProxy : public QObject {
+    Q_OBJECT
+  public:
+    explicit QmlConfigProxy(
+            UserSettingsPointer pConfig,
+            QObject* parent = nullptr);
+
+    Q_INVOKABLE QVariantList getHotcueColorPalette();
```

Comment:
the settings can change. I'd prefer to use QProperties here which can notify the QML on changes to the palette. What do you think? Unforunately its a lot more boilerplate code...

Reply:
Unfortunately, we have no way to listen for changes (becasue the config object doesn't emit signals).I think this can be added later.

Reply:
Right, creating a property without a notify signal is probably a misleading API and might be awkward to use as well. I'm fine with the current solution then.